services/drupal/web/themes/epa_theme/source/_patterns/07-pages/listing-pages/news-releases.yml

```diff
@@ -3,3 +3,144 @@ body_class: 'not-front'
 page_title: 'Search News Releases'
 has_footer: false
 intro: false
+collection:
+  -
+    url: '#'
+    show_calendar: false
+    heading: 'Long Island Scientist Wins EPA Advanced Septic System Nitrogen Sensor Challenge'
+    description: '<strong>News Release Subtitle</strong></p><p class="usa-collection__description">This is the summary, which can contain <abbr title="Hyper Text Markup Language">HTML</abbr> markup. It should be 300 characters or less.'
```

Comment:
@jbwagner It’s bad practice to have something like `</p><p class="usa-collection__description">` passed to a sub-component. It’s better to create a new sub-component that will handle the different use cases.

In this example, creating a new subtitle sub-component in the collection-item component makes this easier to integrate with Drupal and makes maintenance easier in the future.